.github/workflows/main.yml

```diff
@@ -166,7 +166,7 @@ jobs:
       - name: Get current date
         if: always()
         id: date
-        run: echo "::set-output name=date::$(date +'%Y_%m_%dT%H_%M_%S')" #no colons allowed for artifacts
+        run: echo "date=$(date +'%Y_%m_%dT%H_%M_%S') >> $GITHUB_OUTPUT" #no colons allowed for artifacts
```

Comment:
Move the quotes to before you pipe to $GITHUB_OUTPUT

Reply:
Fixed. Thanks!